private/buf/cmd/buf/buf.go

```diff
@@ -97,6 +98,11 @@ const (
 	pushDeprecationMessage                     = `"buf push" has been moved to "buf mod push".` + bufcli.DeprecationMessageSuffix
 )
 
+// Deprecation messages for the hidden commands replaced by the "completion" command
+func shellDeprecationMessage(shell string) string {
```

Comment:
See our Go code standards regarding File Layout, private functions should be below public ones.